fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/model/ProcedureDef.java

```diff
@@ -49,6 +50,8 @@ public void apply(Integer priorVersion, IDatabaseAdapter target) {
         }
 
         // Procedures are applied with "Create or replace", so just do a regular apply
+        // we need to drop and then apply.
```

Comment:
does this comment invalidate the comment on the previous line?

Reply:
Yes, you are right.   Fixed